app/workers/organizations/delete_worker.rb

```diff
@@ -14,11 +14,12 @@ def perform(organization_id, operator_id)
 
       Organizations::Delete.call(org)
 
-      user.touch(:organization_info_updated_at)
-      EdgeCache::BustUser.call(user)
+      if update_and_notify_user
+        user.touch(:organization_info_updated_at)
+        EdgeCache::BustUser.call(user)
 
-      # notify user that the org was deleted
-      NotifyMailer.with(name: user.name, org_name: org.name, email: user.email).organization_deleted_email.deliver_now
+        NotifyMailer.with(name: user.name, org_name: org.name, email: user.email).organization_deleted_email.deliver_now
+      end
```

Comment:
I think that a bit of context described in the comment would be useful in code to make it clearer that we bust cache and notify user if the org is deleted by them. E.g. renaming the variable to reflect it (smth like `deleted_by_org_admin`) or adding a comment.

Reply:
+1 on `deleted_by_org_admin`, although I wonder if it would be worth checking if the user identified by `operator_id` is an org admin within the worker instead of having that passed as a flag?

Reply:
> although I wonder if it would be worth checking if the user identified by operator_id is an org admin within the worker instead of having that passed as a flag?

Initially I was thinking about that and then I remembered that an org admin could be deleting their organization as a _normal user from the user facing side_ which would then branch off incorrectly  in the code and not notify them accordingly etc. Hence, it would be best if we pass the parameter through from the admin controller.

Reply:
Agree that we can't confidently detect the intention by user being an org admin because a user can both be an org admin and a "regular" admin. I thing a parameter makes the intention more explicit.